include/device.h

```diff
@@ -172,6 +172,19 @@ extern "C" {
 			pm_control_fn,					\
 			data_ptr, cfg_ptr, level, prio, api_ptr)
 
+/**
+ * @def DEVICE_DT_INST_DEFINE
+ *
+ * @brief Like DEVICE_DT_DEFINE for an instance of a DT_DRV_COMPAT compatible
+ *
+ * @param inst instance number.  This is replaced by
+ * `DT_DRV_COMPAT(inst)` in the call to DEVICE_DT_DEFINE.
```

Comment:
```suggestion
 * <tt>DT_DRV_INST(inst)</tt> in the call to DEVICE_DT_DEFINE.
```

Reply:
Fixed up the <tt>.  Will leave the re-wording to a followup PR.